tools/smoke-tests/validation-tests/src/test/proto/spine/test/validate/validating_builder_test.proto

```diff
@@ -120,7 +120,25 @@ message Attachment {
     spine.net.Url url = 3 [(required) = true, (valid) = true];
 }
 
+message TaskContents {
```

Comment:
This is a `Book`, an `Article`, or a `Paper`, but not a `Task`, judging on the fields.